### ISSUE TITLE
Doctor search results

### ISSUE THREAD

**Issue by edward-malouf** [codeforboston/jobhopper#379]
The search term Doctor does not bring up "29-1069 Physicians, Surgeons, all other" or "29-1062 Family and General Practitioners" (which I cannot call up regardless)

See screenshot attached.
![Screen Shot 2021-08-02 at 4 40 44 PM](https://user-images.githubusercontent.com/59632463/127922522-29b791fe-784b-4a6d-95ac-97860d140e1d.png)

**Comment by jedpittman** [comment]
I agree, this is strange that doctor doesn't pull the expected results, but
I was able to find the code by searching for "physician"and then select the
code here:
[image: image.png]

I don't think that the site has anything wrong with it; I just thing that
the underlying service is probably not returning the right stuff.
When I look at what the API returns for that query this is the result:
[{"id":194,"soc_code":"29-1071","soc_title":"Physician
Assistants","total_transition_obs":15112.06,"input_match_score":16.66666666666667},{"id":193,"soc_code":"29-1051","soc_title":"Pharmacists","total_transition_obs":23464.72,"input_match_score":33.33333333333333},{"id":206,"soc_code":"29-1123","soc_title":"Physical
Therapists","total_transition_obs":14050.03,"input_match_score":16.66666666666667},{"id":32,"soc_code":"29-1141","soc_title":"Registered
Nurses","total_transition_obs":470309.1,"input_match_score":33.33333333333333}]

That matches exactly what is in the UI. I feel like this used to "work"
when we tried it together...but do we have any idea when it 'stopped'
working? Or, are we sure that it always used to work?
